src/mono/CMakeLists.txt

```diff
@@ -862,6 +862,7 @@ endif()
 ### End of OS specific checks
 
 include_directories("${CLR_SRC_NATIVE_DIR}")
+include_directories("${CLR_SRC_NATIVE_DIR}/public")
```

Comment:
```suggestion
```

don't do this.  add `target_link_libraries (<target> PRIVATE monoapi)` to make includes of the public API headers work.